pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -142,6 +142,10 @@ func (c *cephStatusChecker) checkStatus() {
 		if err := c.forceDeleteStuckRookPodsOnNotReadyNodes(); err != nil {
 			logger.Errorf("failed to delete pod on not ready nodes. %v", err)
 		}
+
+		if status.Health.Checks["OSD_NEARFULL"].Severity == "HEALTH_WARN" || status.Health.Checks["OSD_NEARFULL"].Severity == "HEALTH_ERR" {
+			increaseOSDsCapacity()
+		}
```

Comment:
What if `status.Health.Checks["OSD_NEARFULL"]` does not have content? Using `.Severity` on this can cause a nil value exception.